doc/source/dev/core-dev/github.rst.inc

```diff
@@ -41,26 +41,25 @@ Dealing with pull requests
 --------------------------
 
 - When merging contributions, a committer is responsible for ensuring that
-  those meet the requirements outlined in `Contributing to SciPy
-  <https://github.com/scipy/scipy/blob/main/doc/source/dev/hacking.rst>`_.
-  Also check
-  that new features and backwards compatibility breaks were discussed on the
-  scipy-dev mailing list.
+  those meet the requirements outlined in :ref:`Contributing to SciPy <hacking>`.
```

Comment:
The title at the top of the document is "Ways to Contribute", but I'm not bothered by the difference.